File: jdk/StudentProject_ver3/src/controller/SearchController.java
package controller;

import java.util.Scanner;

import service.StudentService;
import vo.StudentVO;

public class SearchController implements Controller {

	@Override
	public void execute(Scanner sc) {
		System.out.println("학생 정보 검색을 시작합니다..........");
		System.out.print("검색할 학번을 입력하세요 : ");
		String studentNo = sc.nextLine();
		
		StudentVO vo = StudentService.getInstance().searchStudent(studentNo);
		
		if(vo == null)
			System.out.println("학번에 해당하는 학생 정보가 없습니다.");
		else
			System.out.println(vo);
	}

}

File: C#/StudentProject/StudentProject/SearchController.cs
using System;

namespace StudentProject
{
    internal class SearchController : Controller
    {
        public void execute()
        {
            Console.WriteLine("학생정보 검색을 시작합니다.....");
            Console.Write("이름 : ");
            string name = Console.ReadLine();

            StudentVO vo = StudentService.Instance.searchStudent(name);

            if(vo != null)
            {
                Console.WriteLine(vo.ToString());
            }
            else
            {
                Console.WriteLine("검색 결과가 없습니다.");
            }


        }
    }
}